Install error in centos

When I install hictkR, i had a trouble.

`
======== Computing dependency graph ========
hictk/0.0.12: Checking remote: conancenter
bshoshany-thread-pool/4.1.0: Checking remote: conancenter
fast_float/6.1.1: Checking remote: conancenter
fmt/10.2.1: Checking remote: conancenter
hdf5/1.14.3: Checking remote: conancenter
zlib/1.3.1: Checking remote: conancenter
cmake/3.29.6: Checking remote: conancenter
highfive/2.9.0: Checking remote: conancenter
libdeflate/1.19: Checking remote: conancenter
parallel-hashmap/1.3.11: Checking remote: conancenter
span-lite/0.11.0: Checking remote: conancenter
spdlog/1.13.0: Checking remote: conancenter
zstd/1.5.6: Checking remote: conancenter
readerwriterqueue/1.0.6: Checking remote: conancenter
concurrentqueue/1.0.4: Checking remote: conancenter
Graph root
    conanfile.txt: /tmp/Rtmptt6yBK/R.INSTALL213bc34e7770d/hictkR/conanfile.txt
Requirements
    bshoshany-thread-pool/4.1.0#be1802a8768416a6c9b1393cf0ce5e9c - Cache (conancenter)
    concurrentqueue/1.0.4#1e48e1c712bcfd892087c9c622a51502 - Cache (conancenter)
    fast_float/6.1.1#e29acaa3d0543dee343abe3f6815346e - Cache (conancenter)
    fmt/10.2.1#9199a7a0611866dea5c8849a77467b25 - Cache (conancenter)
    hdf5/1.14.3#1d8be3c586fdf68d196b40d6d007d88e - Cache (conancenter)
    hictk/0.0.12#8e413cd45528da38b5a41ccffee41d6d - Cache (conancenter)
    highfive/2.9.0#c57477beed8b0110fadeb6da8f48bcc5 - Cache (conancenter)
    libdeflate/1.19#3ea74a4549efc14d4b1202dc4bfbf602 - Cache (conancenter)
    parallel-hashmap/1.3.11#1e67f4855a3f7cdeb977cc472113baf7 - Cache (conancenter)
    readerwriterqueue/1.0.6#aaa5ff6fac60c2aee591e9e51b063b83 - Cache (conancenter)
    span-lite/0.11.0#519fd49fff711674cfed8cd17d4ed422 - Cache (conancenter)
    spdlog/1.13.0#8e88198fd5b9ee31d329431a6d0ccaa2 - Cache (conancenter)
    zlib/1.3.1#f52e03ae3d251dec704634230cd806a2 - Cache (conancenter)
    zstd/1.5.6#afefe79a309bc2a7b9f56c2093504c8b - Cache (conancenter)
Build requirements
    cmake/3.29.6#1a639e3e7401ea46e16c83ec9c5a7922 - Cache (conancenter)
Resolved version ranges
    cmake/[>=3.18 <4]: cmake/3.29.6
    cmake/[>=3.25 <4]: cmake/3.29.6
    zlib/[>=1.2.11 <2]: zlib/1.3.1

======== Computing necessary packages ========
ERROR: hictk/0.0.12: Error in validate() method, line 77
	if minimum_version and loose_lt_semver(str(self.settings.compiler.version), minimum_version):
while calling 'loose_lt_semver', line 71
	lv1 = [int(v) for v in v1.split(".")]
while calling '<listcomp>', line 71
	lv1 = [int(v) for v in v1.split(".")]
	ValueError: invalid literal for int() with base 10: 'None'
Traceback (most recent call last):
  File "/tmp/Rtmptt6yBK/R.INSTALL213bc34e7770d/hictkR/src/../run_conan.py", line 249, in <module>
    main()
  File "/tmp/Rtmptt6yBK/R.INSTALL213bc34e7770d/hictkR/src/../run_conan.py", line 244, in main
    run_conan_install(conan, env)
  File "/tmp/Rtmptt6yBK/R.INSTALL213bc34e7770d/hictkR/src/../run_conan.py", line 211, in run_conan_install
    sp.check_call(
  File "/root/miniconda3/lib/python3.9/subprocess.py", line 373, in check_call
    raise CalledProcessError(retcode, cmd)
subprocess.CalledProcessError: Command '['/root/miniconda3/bin/conan', 'install', '../conanfile.txt', '--settings=build_type=Release', '--settings=compiler.cppstd=17', '--output-folder=conan-staging', '--build=missing', '--update']' returned non-zero exit status 1.
g++ -std=c++17 -fPIC  -I"/usr/local/lib64/R/include" -DNDEBUG    -I/usr/local/lib64/R/library/Rcpp/include -I'/usr/local/lib64/R/library/Rcpp/include' -I/usr/local/include      -c RcppExports.cpp -o RcppExports.o
g++ -std=c++17 -fPIC  -I"/usr/local/lib64/R/include" -DNDEBUG    -I/usr/local/lib64/R/library/Rcpp/include -I'/usr/local/lib64/R/library/Rcpp/include' -I/usr/local/include      -c hictkr.cpp -o hictkr.o
In file included from hictkr.cpp:10:
./hictkr_file.h:10:10: fatal error: hictk/cooler/cooler.hpp: No such file or directory
 #include <hictk/cooler/cooler.hpp>
          ^~~~~~~~~~~~~~~~~~~~~~~~~
compilation terminated.
make: *** [hictkr.o] Error 1
ERROR: compilation failed for package ‘hictkR’
* removing ‘/usr/local/lib64/R/library/hictkR’
Warning message:
In i.p(...) :
  installation of package ‘/tmp/Rtmp5qaKam/file20ce140716ca1/hictkR_0.0.2.tar.gz’ had non-zero exit status

`

how can i solve it？

ok! There are all log

```
Downloading GitHub repo paulsengroup/hictkR@HEAD
── R CMD build ───────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────
?  checking for file ‘/tmp/RtmpQsqVRK/remotes237cd569a92a5/paulsengroup-hictkR-ef465e3/DESCRIPTION’ ...
─  preparing ‘hictkR’:
?  checking DESCRIPTION meta-information ...
─  cleaning src
─  checking for LF line-endings in source and make files and shell scripts (6.4s)
─  checking for empty or unneeded directories
   Omitted ‘LazyData’ from DESCRIPTION
─  building ‘hictkR_0.0.2.tar.gz’
   
* installing *source* package ‘hictkR’ ...
** using staged installation
** libs
ERROR: Profile '/root/.conan2/profiles/default' already exists
[settings]

arch=x86_64

build_type=Release

os=Linux

compiler=gcc


======== Input profiles ========
Profile host:
[settings]
arch=x86_64
build_type=Release
compiler=gcc
compiler.cppstd=17
os=Linux

Profile build:
[settings]
arch=x86_64
build_type=Release
compiler=gcc
os=Linux


======== Computing dependency graph ========
hictk/0.0.12: Checking remote: conancenter
bshoshany-thread-pool/4.1.0: Checking remote: conancenter
fast_float/6.1.1: Checking remote: conancenter
fmt/10.2.1: Checking remote: conancenter
hdf5/1.14.3: Checking remote: conancenter
zlib/1.3.1: Checking remote: conancenter
cmake/3.29.6: Checking remote: conancenter
highfive/2.9.0: Checking remote: conancenter
libdeflate/1.19: Checking remote: conancenter
parallel-hashmap/1.3.11: Checking remote: conancenter
span-lite/0.11.0: Checking remote: conancenter
spdlog/1.13.0: Checking remote: conancenter
zstd/1.5.6: Checking remote: conancenter
readerwriterqueue/1.0.6: Checking remote: conancenter
concurrentqueue/1.0.4: Checking remote: conancenter
Graph root
    conanfile.txt: /tmp/RtmpsXIvZ3/R.INSTALL2385e2ca3efd6/hictkR/conanfile.txt
Requirements
    bshoshany-thread-pool/4.1.0#be1802a8768416a6c9b1393cf0ce5e9c - Cache (conancenter)
    concurrentqueue/1.0.4#1e48e1c712bcfd892087c9c622a51502 - Cache (conancenter)
    fast_float/6.1.1#e29acaa3d0543dee343abe3f6815346e - Cache (conancenter)
    fmt/10.2.1#9199a7a0611866dea5c8849a77467b25 - Cache (conancenter)
    hdf5/1.14.3#1d8be3c586fdf68d196b40d6d007d88e - Cache (conancenter)
    hictk/0.0.12#8e413cd45528da38b5a41ccffee41d6d - Cache (conancenter)
    highfive/2.9.0#c57477beed8b0110fadeb6da8f48bcc5 - Cache (conancenter)
    libdeflate/1.19#3ea74a4549efc14d4b1202dc4bfbf602 - Cache (conancenter)
    parallel-hashmap/1.3.11#1e67f4855a3f7cdeb977cc472113baf7 - Cache (conancenter)
    readerwriterqueue/1.0.6#aaa5ff6fac60c2aee591e9e51b063b83 - Cache (conancenter)
    span-lite/0.11.0#519fd49fff711674cfed8cd17d4ed422 - Cache (conancenter)
    spdlog/1.13.0#8e88198fd5b9ee31d329431a6d0ccaa2 - Cache (conancenter)
    zlib/1.3.1#f52e03ae3d251dec704634230cd806a2 - Cache (conancenter)
    zstd/1.5.6#afefe79a309bc2a7b9f56c2093504c8b - Cache (conancenter)
Build requirements
    cmake/3.29.6#1a639e3e7401ea46e16c83ec9c5a7922 - Cache (conancenter)
Resolved version ranges
    cmake/[>=3.18 <4]: cmake/3.29.6
    cmake/[>=3.25 <4]: cmake/3.29.6
    zlib/[>=1.2.11 <2]: zlib/1.3.1

======== Computing necessary packages ========
ERROR: hictk/0.0.12: Error in validate() method, line 77
	if minimum_version and loose_lt_semver(str(self.settings.compiler.version), minimum_version):
while calling 'loose_lt_semver', line 71
	lv1 = [int(v) for v in v1.split(".")]
while calling '<listcomp>', line 71
	lv1 = [int(v) for v in v1.split(".")]
	ValueError: invalid literal for int() with base 10: 'None'
Traceback (most recent call last):
  File "/tmp/RtmpsXIvZ3/R.INSTALL2385e2ca3efd6/hictkR/src/../run_conan.py", line 249, in <module>
    main()
  File "/tmp/RtmpsXIvZ3/R.INSTALL2385e2ca3efd6/hictkR/src/../run_conan.py", line 244, in main
    run_conan_install(conan, env)
  File "/tmp/RtmpsXIvZ3/R.INSTALL2385e2ca3efd6/hictkR/src/../run_conan.py", line 211, in run_conan_install
    sp.check_call(
  File "/root/miniconda3/lib/python3.9/subprocess.py", line 373, in check_call
    raise CalledProcessError(retcode, cmd)
subprocess.CalledProcessError: Command '['/root/miniconda3/bin/conan', 'install', '../conanfile.txt', '--settings=build_type=Release', '--settings=compiler.cppstd=17', '--output-folder=conan-staging', '--build=missing', '--update']' returned non-zero exit status 1.
g++ -std=c++17 -fPIC  -I"/usr/local/lib64/R/include" -DNDEBUG    -I/usr/local/lib64/R/library/Rcpp/include -I'/usr/local/lib64/R/library/Rcpp/include' -I/usr/local/include      -c RcppExports.cpp -o RcppExports.o
g++ -std=c++17 -fPIC  -I"/usr/local/lib64/R/include" -DNDEBUG    -I/usr/local/lib64/R/library/Rcpp/include -I'/usr/local/lib64/R/library/Rcpp/include' -I/usr/local/include      -c hictkr.cpp -o hictkr.o
In file included from hictkr.cpp:10:
./hictkr_file.h:10:10: fatal error: hictk/cooler/cooler.hpp: No such file or directory
 #include <hictk/cooler/cooler.hpp>
          ^~~~~~~~~~~~~~~~~~~~~~~~~
compilation terminated.
make: *** [hictkr.o] Error 1
ERROR: compilation failed for package ‘hictkR’
* removing ‘/usr/local/lib64/R/library/hictkR’
Warning message:
In i.p(...) :
  installation of package ‘/tmp/RtmpQsqVRK/file237cd1c5eef77/hictkR_0.0.2.tar.gz’ had non-zero exit status
```

--- a/src/Makevars
+++ b/src/Makevars
@@ -6,9 +6,10 @@ export CXX17 = $(CXX)
 export MACOSX_DEPLOYMENT_TARGET = 10.15
 
 COMPILER_NAME = $(shell "$(CC)" --version | grep -oE '^gcc|clang' | head -n 1)
+FILESYSTEM_LINK_FLAGS = ""
 ifeq "$(COMPILER_NAME)" "gcc"
     FILESYSTEM_LINK_FLAGS = -lstdc++fs
-else
+else ifeq "$(COMPILER_NAME)" "clang"
     FILESYSTEM_LINK_FLAGS = -lc++fs
 endif
 

--- a/src/Makevars
+++ b/src/Makevars
@@ -5,7 +5,7 @@ export CXX = "$(shell "$(R_HOME)/bin/Rscript" -e 'unname(Sys.which("g++"))' | se
 export CXX17 = $(CXX)
 export MACOSX_DEPLOYMENT_TARGET = 10.15
 
-COMPILER_NAME = $(shell "$CC" --version | grep -oE '^gcc|clang' | head -n 1)
+COMPILER_NAME = $(shell "$(CC)" --version | grep -oE '^gcc|clang' | head -n 1)
 ifeq "$(COMPILER_NAME)" "gcc"
     FILESYSTEM_LINK_FLAGS = -lstdc++fs
 else

--- a/src/Makevars
+++ b/src/Makevars
@@ -5,7 +5,7 @@ export CXX = "$(shell "$(R_HOME)/bin/Rscript" -e 'unname(Sys.which("g++"))' | se
 export CXX17 = $(CXX)
 export MACOSX_DEPLOYMENT_TARGET = 10.15
 
-COMPILER_NAME = $(shell "$(CC)" --version | grep -oE '^gcc|clang' | head -n 1)
+COMPILER_NAME = $(shell "$(CC)" --version | grep -oE '^gcc|^clang' | head -n 1)
 FILESYSTEM_LINK_FLAGS = ""
 ifeq "$(COMPILER_NAME)" "gcc"
     FILESYSTEM_LINK_FLAGS = -lstdc++fs

--- a/src/Makevars
+++ b/src/Makevars
@@ -5,6 +5,13 @@ export CXX = "$(shell "$(R_HOME)/bin/Rscript" -e 'unname(Sys.which("g++"))' | se
 export CXX17 = $(CXX)
 export MACOSX_DEPLOYMENT_TARGET = 10.15
 
+COMPILER_NAME = $(shell "$CC" --version | grep -oE '^gcc|clang' | head -n 1)
+ifeq "$(COMPILER_NAME)" "gcc"
+    FILESYSTEM_LINK_FLAGS = -lstdc++fs
+else
+    FILESYSTEM_LINK_FLAGS = -lc++fs
+endif
+
 CONANDEPS_MK := $(shell ../run_conan.py)
 include $(CONANDEPS_MK)
 
@@ -18,3 +25,4 @@ PKG_CPPFLAGS := $(PKG_CPPFLAGS) $(shell "$(R_HOME)/bin/Rscript" -e 'Rcpp:::CxxFl
 PKG_LIBS := $(addprefix -L ,$(CONAN_LIB_DIRS))
 PKG_LIBS := $(PKG_LIBS) $(addprefix -l,$(CONAN_LIBS))
 PKG_LIBS := $(PKG_LIBS) $(addprefix -l,$(CONAN_SYSTEM_LIBS))
+PKG_LIBS := $(PKG_LIBS) $(FILESYSTEM_LINK_FLAGS)

